config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-main-periodics.yaml

```diff
@@ -484,42 +194,3 @@ periodics:
     testgrid-num-failures-to-alert: "4"
     description: Shows test coverage for CAPV
 
-- name: periodic-cluster-api-provider-vsphere-janitor
```

Comment:
we should probably do this in two phases, so the janitor gets a chance to run after the jobs are turned down?

Reply:
Agreed. restored.